README.md

```diff
@@ -168,13 +168,15 @@ cd cri-o
 ### Build
 
 ```bash
+make install.tools
```

Comment:
This is restoring the line removed [earlier today][1] in #1388.  I'm +1 on the restoration.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1388/files#r171273949